src/ProjectManager.cpp

```diff
@@ -124,62 +124,50 @@ void ProjectManager::SaveWindowSize()
       }
       else
          foundIconizedProject =  TRUE;
-
    }
-   if (validWindowForSaveWindowSize)
-   {
-      wxRect windowRect = validProject->GetRect();
-      wxRect normalRect = validProject->GetNormalizedWindowState();
-      bool wndMaximized = validProject->IsMaximized();
+
+   auto SaveWindowPreferences = [](const auto& windowRect, const auto& normalRect, auto isMaximized, auto isIconized) {
```

Comment:
I would prefer not to use auto for argument types.

Reply:
I have replaced the auto parameter types with actual types.

Change the SaveWindowPreferences lambda to a simple function instead.